odoo/addons/base/models/res_users.py

```diff
@@ -232,7 +232,7 @@ def _search_full_name(self, operator, operand):
         where = []
         for group in operand:
             values = [v for v in group.split('/') if v]
-            group_name = values.pop().strip()
+            group_name = values and values.pop().strip()
```

Comment:
IMO, it is weird to have `group_name` is a string or an empty list:
```suggestion
            group_name = values.pop().strip() if values else ""
```
Is it better to have something like that instead to force `group_name` having the same type ?